src/main/kotlin/com/hibob/academy/resource/PetResource.kt

```diff
@@ -43,9 +40,26 @@ class PetResource(
     }
 
     @PUT
-    @Path("/{petId}/{ownerId}")
-    fun assignOwnerIdToPet(@PathParam("petId") petId: UUID, @PathParam("ownerId") ownerId: UUID): Response {
-        return Response.ok(petService.assignOwnerIdToPet(petId, ownerId)).build()
+    @Path("/{petId}/owner/{ownerId}/{companyId}")
```

Comment:
It is better to add info about the param
@Path("/pet/{petId}/owner/{ownerId}/company/{companyId}") also in the next url